### ISSUE TITLE
Ideal implementation of Schema.org's page owner info.

### ISSUE THREAD

**Issue by natejacobson** [washingtonstateuniversity/WSU-spine#17]
Site/page contact and owner information are a required component of university webpages. We display this in the contact tab of the Spine and were providing the info in a schema/microformat friendly way, generating it from a list of meta tags in the head. Turns out that Google needs that data outside the head and structured. I'm lamenting the need to move this out of the head while experimenting with the markup. I haven't entirely given up on the meta approach yet, though it's now moved down to the footer for easy inclusion in template footers. Should it be a child of the Spine instead, as before? Should I just use Google's prescribed spans instead of metas? 

http://www.google.com/webmasters/tools/richsnippets?q=http%3A%2F%2Fnbj.me%2Fspine%2F1%2Fspine.html

The markup:

	<address id="site-contact" itemscope itemtype="http://schema.org/Organization">
		<meta itemprop="name" class="required" content="Washington State University">
		<meta itemprop="department" class="required" content="University Unit">
		<meta itemprop="url" class="required" content="your-unit.wsu.edu">
		<meta itemprop="location" class="optional" content="Your Building">
		<span itemprop="address" itemscope itemtype="http://schema.org/PostalAddress">
			<meta itemprop="streetAddress" class="optional" content="PO Box 555555">
			<meta itemprop="addressLocality" class="optional" content="Pullman, WA">
			<meta itemprop="postalCode" class="required" content="99163">
		</span>
		<meta itemprop="telephone" class="required" content="(509) 335-3564">
		<meta itemprop="email" class="required" content="your-contact-email@wsu.edu">
		<meta itemprop="ContactPoint" title="Contact Page" class="optional" content="http://contact.wsu.edu">
	</address>

And post javascript ...

	<address itemscope itemtype="http://schema.org/Organization" class="hcard">
		<div class="organization-unit fn org"><a href="your-unit.wsu.edu" class="url">University Unit</a></div>
		<div class="organization-name">Washington State University</div>
		<div class="address">
			<div class="street-address">PO Box 555555</div>
			<div class="locality">Pullman, WA <span class="postalcode">99164</span></div>
		</div>
		<div class="tel"><i class="wsu-icon"></i>(509) 335-3564</div>
		<div class="email" rel="email"><a href="mailto:your-contact-email@wsu.edu"><i class="wsu-icon"></i>Email us</a></div>
		<div class="more"><a href="http://contact.wsu.edu"><i class="wsu-icon"></i>Contact Page</a></div>
	</address>

**Comment by wjbonner** [comment]
I wanted to chime in with regards to the W3C Microdata specs.  When switching from meta to span tags the use of the content attribute should have likewise changed to be the elements content (textContent in DOM), at least according to the specs.

http://www.w3.org/TR/2013/NOTE-microdata-20131029/#values

I've noticed with some disappointment that schema.org makes the above mistake in their examples, and googles testing tool doesn't catch this.  However all the other microdata parsers I've seen distinguish how to retrieve the content based on the use of span vs meta.  I'm not sure it is particularly important, but if you are going to spend anytime fretting about this I figured I'd chime in on what the standards prescribe.
